### PR TITLE
Route server

### PR DESCRIPTION
**Pull request by wildcoder51**
We created route server for './family' and './event'.
And we created both files called conf.js and conf-ex.js.

### CODE REVIEW

File: back/index.js [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
 
-app.get('/', (request, response) => {
-  response.send('Welcome to Express');
+
+app.get('/', (req, res) => {
+  res.send('Welcome to Express')
+})
+
+
+app.get('/family', (req, res) => {
```
Comment (rocambille) [inline back/index.js]: check REST naming conventions

File: back/index.js [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+    if (err) {
+      res.status(500).send('Erreur lors de la récupération des familles');
+    } else {
+      res.json(results);
+    }
+  });
 });
 
+
+app.get('/event', (req, res) => {
```
Comment (rocambille) [inline back/index.js]: check REST naming conventions

**Review by rocambille — CHANGES_REQUESTED** [review]

File: back/index.js (hunk 2 above)
Comment (wildcoder51) [reply back/index.js]: Problem has been resolved.
event > events

File: back/index.js (hunk 1 above)
Comment (wildcoder51) [reply back/index.js]: Problem has been resolved.
family > families

**Review by ghost — APPROVED** [review]

**Pull request merged**